packages/amplication-data-service-generator/src/server/static/README.md

```diff
@@ -2,58 +2,84 @@
 
 ## Available Scripts
 
-In the project directory you can run:
+In the `server` subdirectory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.
+Runs the app in development mode.
 By default, it is accessible at http://localhost:3000
 
 ### `npm test`
 
-Launches the test runner.
+Runs tests.
 
 ### `npm run build`
 
-Builds the app for production to the `dist` folder.
+Builds the app for production in the `dist` folder.
 
-The build is minified and the filenames include the hashes.
 Your app is ready to be deployed!
 
-## Development
+## Environment Variables:
 
-Make sure you have Node.js 14, npm, and Docker installed.
+| Environment          | Description                              | Value                                                       |
+| -------------------- | ---------------------------------------- | ----------------------------------------------------------- |
+| DEBUG_MODE           | Debug level                              | 1                                                           |
+| POSTGRESQL_URL       | Local database connection URL            | postgresql://admin:admin@localhost:5432/\${SERVICE_DB_NAME} |
+| POSTGRESQL_PORT      | Local database port                      | 5432                                                        |
+| POSTGRESQL_USER      | Local database username                  | admin                                                       |
+| POSTGRESQL_PASSWORD  | Local database password                  | admin                                                       |
+| COMPOSE_PROJECT_NAME | Docker Compose project name              | amp\_{applicationId}                                        |
+| SERVER_PORT          | The port that the server is listening to | 3000                                                        |
+| JWT_SECRET_KEY       | JWT secret                               | XAFzBpM3es                                                  |
```

Comment:
I dont think we should use a real password - it should be clear to the user that this is an example they need to change - so using something like 123456 or changMe is better